modules/main.nf

```diff
@@ -209,6 +209,23 @@ process index_bam{
     """
 }
 
+process generate_gene_counts4drop{
+
+	input:
+		tuple val(sample), path(counts)
```

Comment:
Wouldn't you need an optional input with external_geneCounts.tsv in order to provide the background database?

Reply:
A good question. In that scenario I'll need to write a function in the script to merge different external_geneCounts.tsv files

Reply:
No, not here. I was thinking gene counts for background db should be a separate tsv file

Reply:
Ah OK